.circleci/tests/testfiles/wiz-kubernetes-integration/cloud.yaml

```diff
@@ -0,0 +1,29 @@
+global:
+  wizApiToken:
+    clientId: clientid
+    clientToken: clientsecret
+
+wiz-kubernetes-connector:
+  enabled: true
+  autoCreateConnector:
+    enabled: true
+    autoDeleteConnectorEnabled: true
+  broker:
+    enabled: true
+
+wiz-sensor:
```

Comment:
Unfortunately, this means that we need to change the sensor bot to also run `task update-golden` in their commits - otherwise this will fail the build

Reply:
That's a very bad idea
Lets do some regex hack to ignore the senor version

Reply:
We can also change their bot to create a PR instead of a direct commit to master, and have them auto-generate the golden files.
In both cases, we can't keep it like it is now, since it's broken - when they activate the bot and create the direct commit to master, without noticing, they will break the build.
(This already happened BTW and that's why we removed the kubernetes-integration test)

Reply:
I think that if we change the bot to create a PR **with the golden files** and have a reviewer go over it and complete the PR, it will work, no?

Reply:
i have no idea about this update-golden thing... ah it's some new test suite?

Reply:
Enable this needs changing the sensor chart release flow and some devops work - created a ticket: https://wiz-io.atlassian.net/browse/WZ-59163
For now this test will be skiped.